packages/create-snowpack-app/index.js

```diff
@@ -124,15 +127,18 @@ if (requiredVersion < 10) {
     path.join(targetDirectory, "package.json"),
     `{"name": "my-csa-app"}`
   );
-  try {
-    await execa("npm", ["install", template, "--ignore-scripts"], {
-      cwd: targetDirectory,
-      all: true,
-    });
-  } catch (err) {
-    // Only log output if the command failed
-    console.error(err.all);
-    throw err;
+  // fetch from npm or GitHub if not local (which will be most of the time)
+  if (!isLocalTemplate) {
+    try {
+      await execa("npm", ["install", template, "--ignore-scripts"], {
+        cwd: targetDirectory,
+        all: true,
+      });
+    } catch (err) {
+      // Only log output if the command failed
+      console.error(err.all);
+      throw err;
+    }
```

Comment:
Reverted the original code here, but now we conditionally run it if a template isn’t local.